base/common/python/pki/__init__.py

```
@@ -443,7 +440,8 @@ def handler(inst, *args, **kwargs):
                     # json raises ValueError. simplejson raises
                     # JSONDecodeError, which is a subclass of ValueError.
                     # re-raise original exception
-                    six.reraise(exc_type, exc_val, exc_tb)
+                    logger.debug('re-raise original exception ignoring exc_type: %s', exc_type)
+                    raise exc_val.with_traceback(exc_tb)
```

Ruff (0.15.2)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)